src/tasks/ruuvi_task_adc.h

```diff
@@ -55,6 +55,8 @@
 #include "ruuvi_driver_sensor.h"
 #include "ruuvi_interface_log.h"
 
+typedef float adc_float;
```

Comment:
Please do not redefine float, it only serves to obscure that actual type is system float